bot/resources/tags/empty_json.md

```diff
@@ -0,0 +1,24 @@
+When creating a new JSON file you might run into the following error.
+
+`JSONDecodeError: Expecting value: line 1 column 1 (char 0)`
+
+In short, this means that your JSON is invalid in its current state. This could very well happen because the file is just new and empty.
+A JSON may never be completely empty. It is recommended to have at least one of the following in your json:
+
+```
+object
+array
+```
```

Comment:
No need to use a list when there are only two items. Furthermore, I don't like using the term "recommend" here. I suggest rephrasing it as "You probably want to use an object or an array."